README.md

```diff
@@ -1,34 +1,26 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
```

Comment:
This should probably be kept. It defines what EIP stands for (which is then used throughout the document) and it gives a pithy introduction.  Did you have a reason for removing it?

Reply:
Thank you, added back in fbe543c79627260d5c10ab8cbd40abca693a03bd